wg-api/spec-documents/api-spec-template.md

```diff
@@ -0,0 +1,16 @@
+# API Spec Template
+
+## Summary
+<!--- 1-3 sentence description of what is changing and why. --->
+
+## Platforms
+<!--- What platforms will be affected? Options include macOS, Windows, Linux, or any combination thereof. --->
+
+## Impetus
```

Comment:
Really glad that this is its own section; I think this is important for justification.